pootle/static/js/editor/app.js

```diff
@@ -2091,8 +2094,17 @@ PTL.editor = {
         $(data.timeline).hide().prependTo('#extras-container')
                         .slideDown(1000, 'easeOutQuad');
       }
-
-      helpers.updateRelativeDates();
+      $('.js-mount-timesince').each((index, el) => {
+        const props = {
+          title: data.entries_group[index].display_datetime,
+          dateTime: data.entries_group[index].iso_datetime,
+        };
+        if (data.entries_group[index].via_upload) {
+          ReactDOM.render(<UploadTimeSince {...props} />, el);
```

Comment:
Note that these components are never unmounted, they remain alive after unit navigation and they will leak memory.